Add a FAQ for HTTPS CORS requests failing because of localhost cert errors

justingrant opened the pull request:
I ran into a problem where CORS was failing due to HTTPS errors on localhost. The root cause was that my localhost web server *and* my localhost API server both needed certificate exceptions added. This commit adds a section to the FAQ explaining how to fix these errors in Firefox, mostly so that people who are searching Google for the error code will find an answer somewhere.

spenibus commented:
Looks helpful.

Will be merged as soon as this is fixed:

>both **an** web server

justingrant committed:
Add a FAQ for handling localhost cert errors

I ran into a problem where CORS was failing due to HTTPS errors. This
commit adds a section to the FAQ explaining how to fix these errors.

justingrant commented:
> as soon as this is fixed:
> 
> > both **an** web server

Oops!  Fixed now.
